.github/workflows/hybridse-ci.yml

```diff
@@ -85,29 +85,31 @@ jobs:
           path: |
             build/hybridse/**/*.xml
 
-  publish-test-results:
-    runs-on: ubuntu-latest
-    needs: ["linux-build", "macos-build"]
+  publish-test-results-linux:
+    needs: ["linux-build"]
     # the action will only run on 4paradigm/HybridSE's context, not for fork repo or dependabot
     if: >
       always() && github.event_name == 'push' || (
       github.event.pull_request.head.repo.full_name == github.repository &&
       github.event.sender.login != 'dependabot[bot]' )
-    steps:
-      - name: Download Artifacts
-        uses: actions/download-artifact@v2
 
-      - name: Publish Linux UT Results
-        uses: EnricoMi/publish-unit-test-result-action@v1
-        with:
-          files: linux-ut-result-*/**/*.xml
-          check_name: HybridSE Linux Test Report
-          comment_title: HybridSE Linux Test Report
-
-      - name: Publish Mac UT Results
-        uses: EnricoMi/publish-unit-test-result-action@v1
-        with:
-          files: macos-ut-result-*/**/*.xml
-          check_name: HybridSE Mac Test Report
-          comment_title: HybridSE Mac Test Report
+    name: Publish Linux UT Results
+    uses: ./.github/workflows/publish-test-results-action.yml
+    with:
+      files: linux-ut-result-*/**/*.xml
+      check_name: HybridSE Linux Test Report
+      comment_title: HybridSE Linux Test Report
 
+  publish-test-results-mac:
+    needs: ["macos-build"]
+    # the action will only run on 4paradigm/HybridSE's context, not for fork repo or dependabot
+    if: >
+      always() && github.event_name == 'push' || (
+      github.event.pull_request.head.repo.full_name == github.repository &&
+      github.event.sender.login != 'dependabot[bot]' )
+    name: Publish Mac UT Results
+    uses: EnricoMi/publish-unit-test-result-action@v1
```

Comment:
Referring to your patch, I removed the `steps`. But when I pushed this changes to my remote repository, I got the following error:

![image](https://user-images.githubusercontent.com/52828870/198546688-dc0ffbaa-42b4-4a24-b98c-fd6b60873a01.png)

Do you encounter the same problem?

Reply:
Note it is not same 'uses' value as above

Reply:
emmmmm, I don't quite understand.

Reply:
> Note it is not same 'uses' value as above line 97

Yes, I konw, reusable workflows should be referenced at the top-level `jobs.*.uses` key. So should I add `runs-on` and `steps` as before when use `EnricoMi/publish-unit-test-result-action@v1`?

Reply:
see https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow. There is no filename in the end. `EnricoMi/publish-unit-test-result-action@v1` is an action, not reusable workflow.